packages/SwingSet/src/kernel/kernel.js

```diff
@@ -10,15 +10,14 @@ import { insistKernelType, parseKernelSlot } from './parseKernelSlots';
 import { parseVatSlot } from '../parseVatSlots';
 import { insistStorageAPI } from '../storageAPI';
 import { insistCapData } from '../capdata';
-import { insistMessage } from '../message';
+import { insistMessage, insistVatDeliveryResult } from '../message';
```

Comment:
I'd feel better about a refactor this big with @ts-check in kernel.js. But I suppose test coverage is pretty good.